test/__snapshots__/test.js.snap

```diff
@@ -6,9 +6,15 @@ exports[`generation built-in types Function 1`] = `
 const conversions = require(\\"webidl-conversions\\");
 const utils = require(\\"./utils.js\\");
 
-exports.convert = (value, { context = \\"The provided value\\" } = {}) => {
+exports.convert = (
+  value,
+  {
+    context = \\"The provided value\\",
+    ctorRegistry = { \\"%TypeError%\\": TypeError, \\"%Number%\\": Number, \\"%Promise%\\": Promise, \\"%String%\\": String }
```

Comment:
Hmm. I think it'd be a bit more uniform for consumers if the convert algorithm took a mandatory first-parameter global, similar to `create()` and friends.